adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -137,6 +137,37 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] extends Logging {
    */
   def replaceSequences(newSequences: SequenceDictionary): U
 
+  /**
+   * Caches underlying RDD in memory.
+   *
+   * @return U cached GenomicRDD
+   */
+  def cache(): U = {
+    rdd.cache()
+    replaceRdd(rdd)
+  }
+
+  /**
+   * Persists underlying RDD in memory.
+   *
+   * @param sl new StorageLevel
+   * @return U persisted GenomicRDD
+   */
+  def persist(sl: StorageLevel): U = {
+    rdd.persist(sl)
+    replaceRdd(rdd)
+  }
+
+  /**
+   * Unpersists underlying RDD from memory.
+   *
+   * @return U uncached RDD
+   */
+  def unpersist(): U = {
+    rdd.unpersist()
+    replaceRdd(rdd)
```

Comment:
@fnothaft it returns U but idk if there is a better way to return the new rdd

Reply:
prefer `replaceRdd(rdd.unpersist())`